plugins/woocommerce/package.json

```diff
@@ -155,7 +132,7 @@
 					]
 				},
 				{
-					"name": "PHP WP: latest - 1",
+					"name": "PHP: 7.4 WP: latest - 1",
```

Comment:
I wouldn't include the PHP version in the name if using the default, we'll probably forget to update it whenever the default version changes.